build.gradle

```diff
@@ -130,6 +130,16 @@ tasks.withType(FindBugs) {
   }
 }
 
+jacocoTestReport {
```

Comment:
There is task for that: `./gradlew jacocoTestReport` (it reads the JaCoCo-produced binary file and outputs the nice HTML reports).